myStyles.css

```diff
@@ -0,0 +1,216 @@
+  .mainPage h1 {
+    font-size: 56px;
+  }
+
+  .mainPage h2 {
+    font-size: 24px;
+  }
+
+  .mainPage p {
+    font-size: 16px;
+  }
+}
+
+@media (min-width: 376px) and (max-width: 768px) {
+  .socialLinks_Bottom {
+    display: block;
+    padding-bottom: 55%;
+  }
+
+  .contentContainer {
+    display: block;
+    max-width: 327px;
+    padding-top: 35%;
+  }
+
+  .mainPage h1 {
+    font-size: 56px;
+  }
+
+  .mainPage h2 {
+    font-size: 24px;
+  }
+
+  .mainPage p {
+    font-size: 16px;
+  }
+}
+
+@media (min-width: 769px) {
+  .flexContainer {
+    /* flex-direction: row; */
+    align-items: center;
+  }
+
+  .contentContainer {
+    display: inline-block;
+    max-width: 530px;
+    padding-top: 15%;
+  }
+
+  .mainPage h1 {
+    font-size: 70px;
+  }
+
+  .mainPage h2 {
+    font-size: 30px;
+  }
+
+  .mainPage p {
+    font-size: 20px;
+  }
+}
+
+.topBar {
+  position: relative;
+  display: flex;
+  width: 100%;
+  left: 0;
+  top: 0;
+  flex-direction: column;
+}
+
+.status_TopNav {
+  position: static;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  background-color: rgb(69, 67, 67);
+  padding-top: 5%;
+  padding-bottom: 3%;
+}
+
+.timeContainer {
+  margin-left: 12%;
+  margin-bottom: 2%;
+}
+
+.statusContainer {
+  margin-right: 4%;
+}
+
+.logoAngostoContainer {
+  margin-left: 8%;
+}
+
+.menuAngosto_TopNav {
+  position: static;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  background-color: rgb(69, 67, 67);
+  padding-top: 1.5%;
+  padding-bottom: 5%;
+}
+
+.homeLogo_Angosto {
+  text-decoration: none;
+  color: aliceblue;
+  font-size: 25px;
+  font-weight: bold;
+}
+
+#baraMenuImage {
+  align-self: center;
+}
+
+.menuHamburgerContainer {
+  margin-right: 10%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.flexContainer {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.mainPage {
+  position: relative;
+  display: block;
+  padding-left: 8%;
+  padding-right: 8%;
+  background-color: black;
+}
+
+.mainPage h1 {
+  color: #ff6b00;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+}
+
+.mainPage h2 {
+  color: #fff5e1;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  font-family: 'Roboto', sans-serif;
+  font-style: bold;
+  font-weight: 700;
+  line-height: 32px;
+}
+
+.mainPage p {
+  color: #fff;
+  font-family: 'Poppins', sans-serif;
+}
+
+.background {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  height: auto;
+  z-index: -1;
+  background-repeat: repeat-x;
+}
+
+.socialLinks_Bottom {
+  width: 100%;
+}
+
+.horizontal-list {
+  display: flex;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.horizontal-list li a {
+  display: block;
+  padding: 10px 15px; /* Adjust the padding as needed */
+  text-decoration: none;
+  background-color: #f2f2f2; /* Adjust the background color as needed */
```

Comment:
- [OPTIONAL] Kindly remove this background colour because according to the design there is no background color for the social media icons. By removing the background your design will look good and professional 🚀